### ISSUE TITLE
Rails 6.1 NameError: undefined method `render_partial' for class `ActionView::PartialRenderer'

### ISSUE THREAD

**Issue by aliismayilov** [appoptics/appoptics-apm-ruby#112]
# Description

A rails application that includes appoptics_apm and has `AppOpticsAPM::Config[:action_view][:enabled] = true` is failing to boot with the following output:
```
...
I, [2021-01-21T15:04:01.285126 #54]  INFO -- : [appoptics_apm/loading] Instrumenting actionview
rails aborted!
NameError: undefined method `render_partial' for class `ActionView::PartialRenderer'
/app/config/environment.rb:5:in `<main>'
```

# How to reproduce or use case

1. Start a new rails app.
2. Set up appoptics_apm.
3. Ensure `AppOpticsAPM::Config[:action_view][:enabled] = true` enabled.
4. Eager load the app.

# OS, Ruby version, Gemfile, Gemfile.lock, log message ... (if relevant)
actionview (6.1.1)
appoptics_apm (4.12.1)
Ruby 2.7.2

**Comment by prettycoder** [comment]
thanks for reporting this
Looks like PartialRenderer was heavily refactored between 6.0 and 6.1
I'll adapt the appoptics_apm gem and release a fix shortly

**Comment by sajjadmurtaza** [comment]
@prettycoder we also facing the same issue. Any update?

**Comment by prettycoder** [comment]
sorry, for the delay. Hope to get to it this week.

**Comment by prettycoder** [comment]
fixed in   appoptics_apm-4.12.2.gem
